post-merge update: networks page (#7191)
ethereum/ethereum-org-website: src/content/developers/docs/networks/index.md

You should test any contract code you write on a testnet before deploying to Mainnet. Among dapps that integrate with existing smart contracts, most projects have copies deployed to testnets.

Most testnets started by using a proof-of-authority consensus mechanism. This means a small number of nodes are chosen to validate transactions and create new blocks – staking their identity in the process. Alternatively, some testnets started off using a proof-of-work consensus mechanism with just a few permissioned miners. However, in preparation for [The Merge](/upgrades/merge), these testnets underwent their own transitions to proof-of-stake, offering the opportunity for multiple 'dress-rehearsals' before developers merged Ethereum Mainnet. The Ethereum testnets are now proof-of-stake, just like Ethereum Mainnet.

ETH on testnets has no real value; therefore, there are no markets for testnet ETH. Since you need ETH to actually interact with Ethereum, most people get testnet ETH from faucets. Most faucets are webapps where you can input an address which you request ETH to be sent to.

#### Sepolia {#sepolia}

> Originally a proof-of-work testnet, now proof-of-stake. Sepolia is still running but it might not be maintained long term.

Reviewer: ```suggestion
Sepolia is a proof-of-stake testnet. Although Sepolia is still running, it is not currently planned to be maintained long-term. Before undergoing The Merge in June 2022, Sepolia was a proof-of-work testnet.
```

Reviewer 2: I don't know where this came from but it's factually wrong. Sepolia will be maintained until at least 2026.

https://ethereum-magicians.org/t/proposal-predictable-ethereum-testnet-lifecycle/11575/5

> #### Goerli {#goerli}

Reviewer: Let's put Goerli top of this list if it is expected to be the main testnet people use.

> Originally a proof-of-authority testnet, now proof-of-stake. Goerli is expected to be maintained long term as a stable testnet for application developers.

Reviewer: We are burying the lede here

```suggestion
Goerli is a proof-of-stake testnet. It is expected to be maintained long-term as a stable testnet for application developers. Before its testnet merge, Goerli was a proof-of-authority testnet.
```

> _Note, [the Ropsten testnet is deprecated](https://github.com/ethereum/pm/issues/460) and will no longer receive protocol upgrades. Please consider migrating your applications to Sepolia or Goerli._

Reviewer: Ropsten IS deprecated? Or will be deprecated.

Author: already deprecated (i.e. no active maintenace), will be shut down in Q4

Reviewer: Ah, the language isn't clear here then. We're using deprecated to mean both: 

- no longer maintained
- no longer accessible

> Ropsten was a proof-of-work testnet that went through The Merge to proof-of-stake in May 2022. It is likely to deprecated soon.

Reviewer: ```suggestion
Ropsten is a proof-of-stake testnet. Ropsten will be deprecated in late 2022. Before undergoing The Merge in May 2022, Ropsten was a proof-of-work testnet.
```
